nifi-nar-bundles/nifi-network-bundle/nifi-network-utils/src/main/java/org/apache/nifi/processors/network/parser/Netflowv5Parser.java

```diff
@@ -29,6 +32,7 @@
     private static final int SHORT_TYPE = 0;
     private static final int INTEGER_TYPE = 1;
     private static final int LONG_TYPE = 2;
```

Comment:
I hate to ask, but is there no IPV 6 that could be in there?

Reply:
to my knowledge, there is no IPV6 format in the parsed output.

Reply:
So netflow5 doesn't support IPV6? If not then cool

Reply:
I did some research, it doesn't support it in this version.